source/actions.js

```diff
@@ -162,35 +167,48 @@ export const watchEvent = (firebase, dispatch, event, path, dest, onlyLastEvent
   }
 
   const runQuery = (q, e, p) => {
+      dispatch({
+        type: START,
+        timestamp: moment(),
+        requesting : true,
+        requested : false,
```

Comment:
What is the purpose of requested here? It might be a little more clear to just have one parameter like `isLoading` or `isRequesting`.